### PR TITLE
ref(docs): clean up quickstart documentation

### PR DESCRIPTION
**Pull request by bacongobbler**
Moving some stuff around. Specifically:

 - "Configuring DNS" should now occur after the platform is installed so users can
   deal with it once the routers are up
 - the load balancer configuration was all duplicated information (Managing Deis
   covers this), so it was moved/removed
 - DNS configuration was moved over to "Configuring DNS"
 - removed "Next Steps" from "Installing the Deis Platform" as the next steps are
   described in the Quickstart doc
 - removed the documentation around "Set up the Deis Router". This is no longer necessary
   for a "Quick Start" and should go into a "Configuring Router" document in the future (see deis/router#67)

### CODE REVIEW

**Comment by bacongobbler** [comment]
also dependent on https://github.com/deis/charts/pull/52

**Comment by arschles** [comment]
:+1: for the greatly simplified setup (and lots of red)

File: docs/src/installing-deis/installing-the-deis-platform.md [hunk 1]
```diff
@@ -71,137 +55,6 @@ $ kubectl get pods --namespace=deis
 
 Once you see all of the pods in the `READY` state, your Deis platform is running on a cluster!
 
-## A Note on Routing Traffic
```
Comment (slack) [inline docs/src/installing-deis/installing-the-deis-platform.md]: This should have made it to deis/router#67, no?

File: docs/src/managing-deis/configuring-dns.md [hunk 2]
```diff
@@ -18,16 +18,67 @@ Apps can then be accessed by browsers at `appname.myapps.com`, and the controlle
...
+This will get you started, though you may find that you have to manually maintain this file as you
+create and deploy more applications to the cluster.
+
+## Testing
+
+To test that traffic reaches its intended destination, a request can be
+sent to the Deis controller like so:
+
+```
+curl http://deis.<platform domain>/v2/
```
Comment (slack) [inline docs/src/managing-deis/configuring-dns.md]: With this change, this is the first time `platform domain` is referenced in this doc. Should be `example.com` I think.

File: docs/src/installing-deis/installing-the-deis-platform.md (hunk 1 above)
Comment (bacongobbler) [inline docs/src/installing-deis/installing-the-deis-platform.md]: couldn't. Two separate projects = two separate PRs

File: docs/src/managing-deis/configuring-dns.md (hunk 2 above)
Comment (bacongobbler) [inline docs/src/managing-deis/configuring-dns.md]: this was a tie-over from the old docs (see https://github.com/deis/workflow/pull/215/files#diff-3122c85f28ce9c670a6d843bc29638feL191). I'm more than happy to change it to example.com though :)

**Comment by bacongobbler** [comment]
okay I addressed your comments, @slack. Should be good to go!

**Pull request merged**